docs/user-guide/federation/index.md

```diff
@@ -2,15 +2,68 @@
 title: Federation User Guide
 ---
 
-This guide explains how we can manage multiple kubernetes clusters using
+This guide explains why and how we can manage multiple kubernetes clusters using
 federation.
-[Federation proposal](https://github.com/kubernetes/kubernetes/blob/{{page.githubbranch}}/docs/proposals/federation.md)
-details the use cases motivating cluster federation.
 
 
 * TOC
 {:toc}
 
+
+## Why federation
+
+Federation makes it easy to manage multiple clusters. It does so by providing 2
+major building blocks:
+
+  * Sync resources across clusters: Federation provides the ability to keep
+    resources in multiple clusters in sync. If you want to ensure that a
+    resource exists in multiple clusters, use federation.
+  * Cross cluster discovery: It provides ability to auto-configure DNS servers
+    and load balancers with backends from all clusters. If you want a global VIP
+    or DNS record to access backends from all clusters, use federation.
+
+Some other use cases that federation enables are:
+
+* High Availability: By spreading load across clusters and auto configuring DNS
+  servers and load balancers, federation minimises the impact of cluster
+  failure.
+* Avoiding provider lock-in: By making it easier to migrate application across
```

Comment:
Yes but I wanted to mention it in both sections.
Even HA can be argued to be similar to Fault isolation.